Formula/osgeo-gdal.rb

```diff
@@ -142,7 +142,7 @@ def configure_args
       "--with-qhull=internal",
       "--with-libz=#{Formula["libzip"].opt_prefix}",
       "--with-png=#{Formula["libpng"].opt_prefix}",
-      "--with-libtiff=#{Formula["libtiff"].opt_prefix}",
+      "--with-libtiff=internal", # #{Formula["libtiff"].opt_prefix}
       "--with-geotiff=#{Formula["osgeo-libgeotiff"].opt_prefix}",
```

Comment:
@fjperini I've tried similar thing yesterday but then I had problem with TiffTags. Using external libgeotiff and internal libtiff might create some side effects

Reply:
@vincentsarago can you report the errors? suggestions?

Reply:
Well basically with this configuration I couldn't read any GeoTIFF thus I removed `--with-geotiff` and it seemed to fix it

Reply:
I guess it's better to use both libtiff and libgeotiff shipped with GDAL to avoid conflict

Reply:
maybe @palmerj could validate this

Reply:
yes, maybe we need to use "internal" in both.

Reply:
Thanks both. I've not had any issues using external geotiff and internal libtiff, but I haven't done wide testing. Seem logical to use internal for both.